Automattic/woocommerce-payments
# Delete `client/disputes/details`

Partially fixes https://github.com/Automattic/woocommerce-payments/issues/7363

Note: this PR's branch is based of https://github.com/Automattic/woocommerce-payments/pull/7456's branch.

#### Changes proposed in this Pull Request

- Delete `client/disputes/details` as it's not used anywhere anymore.
- Clean up `client/disputes/style.scss`.

In `client/disputes/details`, there are 2 components:
- [LegacyDisputeDetails](https://github.com/Automattic/woocommerce-payments/blob/a4cded7f89e90f274468888a2878dbd9b927366d/client/disputes/details/index.tsx), and
- [Actions](https://github.com/Automattic/woocommerce-payments/blob/a4cded7f89e90f274468888a2878dbd9b927366d/client/disputes/details/actions.tsx).

For each, I tried to find imports ([#](https://github.com/Automattic/woocommerce-payments/blob/a4cded7f89e90f274468888a2878dbd9b927366d/client/disputes/details/index.tsx#L13-L24) and [#](https://github.com/Automattic/woocommerce-payments/blob/a4cded7f89e90f274468888a2878dbd9b927366d/client/disputes/details/actions.tsx#L13-L14)) that are only used by them, so I could delete. Can't find any.

I also looked for any `className` so I could clean them up from `client/disputes/style.scss`. I found some:
- `wcpay-dispute-details` (used by `DisputeEvidencePage`, so not deleting), and
- `header-dispute-overview` (not referenced anywhere after `LegacyDisputeDetails` is deleted).

#### Testing instructions

Affected client/disputes/evidence/index.js
Make sure imports from `client/disputes/details/index.tsx` are used by others, so we don't need to delete.
Make sure imports from `client/disputes/details/actions.tsx` are used by others, so we don't need to delete.

* Make sure tests are passing.
* Make sure dispute flows are still working.

Since `client/disputes/evidence/index.js` is the most affected component, make sure the UX and look for the dispute challenge form is not affected by this PR.
* Purchase a product with `4000000000000259` so it triggers a dispute.
* Go to Payments > Transactions > click on the transaction for the new purchase.
* Click on the 'Submit evidence'.
* You will be on the [dispute challenge page](https://github.com/Automattic/woocommerce-payments/blob/a4cded7f89e90f274468888a2878dbd9b927366d/client/index.js#L174-L175).
* Compare base branch and PR branch and make sure that page has the same UI and UX.

-------------------


**Post merge**
